andela/ah-titans
# #159053962 Restrict User Rating

#### What does this PR do?
Restricts the user from rating an article more than 5 times and fix ratings test bugs

#### Description of tasks to be completed
1. Get user details to restrict count of ratings
2. Get average rating for an article
3. Fix test bugs

#### How should this be manually tested?
Clone the repo with the following command on your terminal
```
git clone https://github.com/andela/ah-titans/.git
```
Install your virtual environment and activate it then pip install the requirements with the commands:
```
virtualenv venv
```
```
source env/bin/activate
```
```
pip install -r requirements.txt
```
Run the server
```
python manage.py runserver
```
If a user has rated an article more than five times, Postman should display this:

<img width="1000" alt="screen shot 2018-08-07 at 11 00 45" src="https://user-images.githubusercontent.com/29487740/43762732-65bea152-9a31-11e8-8fd1-ba8dcf9e9a7f.png">

 #### Any background context you want to provide?
None

#### What are the relevant Pivotal Tracker Stories?
[#159053962](https://www.pivotaltracker.com/story/show/159053962)
